SU2_CFD/include/numerics_structure.hpp

```diff
@@ -2475,7 +2481,7 @@ class CUpwSca_TurbSST : public CUpwScalar {
   /*!
    * \brief Adds any extra variables to AD
    */
-  void ExtraADPreaccIn();
+  inline void ExtraADPreaccIn() {}
```

Comment:
The "inline" here is redundant.  Functions defined inside a class definition are automatically "inline."

Reply:
In common practice yes but apparently that is not part of the [standard](https://stackoverflow.com/questions/9192077/is-inline-implicit-in-c-member-functions-defined-in-class-definition) (which I have not read) and so some compilers might complain about multiple definition.

Reply:
Ah ok.  So no harm done in leaving it in.